<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonAcessarInicio">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Acessar Inicio</string>
    </property>
    <property name="icon">
     <iconset resource="photos.qrc">
      <normaloff>:/new/files/lupa.png</normaloff>:/new/files/lupa.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRetirarInicio">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Retirar Inicio</string>
    </property>
    <property name="icon">
     <iconset resource="photos.qrc">
      <normaloff>:/new/files/saida.png</normaloff>:/new/files/saida.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Inserir Inicio</string>
    </property>
    <property name="icon">
     <iconset resource="photos.qrc">
      <normaloff>:/new/files/inserir.png</normaloff>:/new/files/inserir.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditValor">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Valor:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Inserir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Acesssar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAcessarFinal">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Acessar Final</string>
    </property>
    <property name="icon">
     <iconset resource="photos.qrc">
      <normaloff>:/new/files/lupa.png</normaloff>:/new/files/lupa.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Retirar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRetirarFinal">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Retirar Final</string>
    </property>
    <property name="icon">
     <iconset resource="photos.qrc">
      <normaloff>:/new/files/saida.png</normaloff>:/new/files/saida.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInsertLast">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Inserir Fim</string>
    </property>
    <property name="icon">
     <iconset resource="photos.qrc">
      <normaloff>:/new/files/inserir.png</normaloff>:/new/files/inserir.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditElements">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quantidade de Elementos:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMe">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="photos.qrc">:/new/files/myPhoto.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditExit">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>269</y>
      <width>491</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Posicao:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRetirarPosicao">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Retirar Posicao</string>
    </property>
    <property name="icon">
     <iconset resource="photos.qrc">
      <normaloff>:/new/files/saida.png</normaloff>:/new/files/saida.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAcessarPosicao">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Acessar Posicao</string>
    </property>
    <property name="icon">
     <iconset resource="photos.qrc">
      <normaloff>:/new/files/lupa.png</normaloff>:/new/files/lupa.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInsertPosicao">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Inserir Posicao</string>
    </property>
    <property name="icon">
     <iconset resource="photos.qrc">
      <normaloff>:/new/files/inserir.png</normaloff>:/new/files/inserir.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="photos.qrc"/>
 </resources>
 <connections/>
</ui>
